packages/light-client/src/utils/utils.ts

```diff
@@ -74,3 +74,7 @@ export function isEmptyHeader(header: BeaconBlockHeader): boolean {
   const emptyValue = ssz.phase0.BeaconBlockHeader.defaultValue();
   return ssz.phase0.BeaconBlockHeader.equals(emptyValue, header);
 }
+
+// Thanks https://github.com/iliakan/detect-node/blob/master/index.esm.js
+export const isNode =
+  Object.prototype.toString.call(typeof process !== "undefined" ? process : 0) === "[object process]";
```

Comment:
should we do this determination by inspecting if window object is present? we inject process in eth2-light-client

Reply:
I used the same logic we used in the `bls` library. There could be different ways just tried to be consistent. 

https://github.com/ChainSafe/bls/blob/710e7f9f5e5c9ab79ac08460a3d7d5a937b379f1/src/getImplementation.ts#L4